src/styles/card.scss

```
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $__color-border-card;
+  border-radius: 5px;
+  max-width: $__card-with;
+  padding: 32px 16px 16px;
+
+  &__img {
+    margin: 0 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    color: $__color-fonts-card;
+  }
+
+  &__product {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $__color-code-price;
+  }
+
+  &__stars {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    color: $__color-fonts-card;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    color: $__color-code-price;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $__color-fonts-card;
+  }
+
+  &__button {
+    padding: 12px 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: $__color-hover-active-button;
```

```suggestion
    background-color: $__color-hover-active-button;
```
Specify the property if you are changing only color